charts/geoserver/README.md

```diff
@@ -91,6 +91,11 @@ $ helm install --set persistence.existingClaim=PVC_NAME geoserver
 
 ## ChangeLog
 
+### 1.1.0
+
+- Added extra configuration options
```

Comment:
make sure to make more clear
also mention you changed http -> geoserver

Reply:
updated chagelog